layouts/wallet.vue

```diff
@@ -3,7 +3,9 @@
     <template v-if="currentAddress">
       <AppHeader />
       <Nuxt class="min-h-full" />
-      <AppFooter/>
+      <AppFooter
+        :class="{'hidden': $nuxt.$route.path.includes('/nft/purchase/')}"
```

Comment:
```suggestion
        :class="{ hidden: $route.path.includes('/nft/purchase/')}"
```

Reply:
why not use name, or do we expect subpath under `/nft/purchase`? But in that case we can still use `name.includes`